src/containers/flows/AcceptBid/components/Details/Details.tsx

```diff
@@ -63,9 +63,12 @@ const Details = ({
 	///////////////
 	const formattedHighestBidAmount = formatBidAmount(
 		highestBid,
-		paymentTokenInfo.name,
+		paymentTokenInfo.symbol,
+	);
+	const formattedBidAmount = formatBidAmount(
```

Comment:
<img width="899" alt="Screenshot 2022-07-14 at 23 04 47" src="https://user-images.githubusercontent.com/39112648/179097067-42c9e393-c2d9-443c-817b-b3b0f25a8fe9.png">

Opening AcceptBid from owned domains table row

Reply:
Unable to reproduce it anymore. Can you check if you're able to reproduce it?